src/comm_droplet.F

```diff
@@ -813,11 +813,12 @@ subroutine GatherIndices(dir,ptrDepart,pdata_neighbor,Depart_ind)
         !$acc end parallel
 
         ! special treament for the last value in the array
-        !$acc serial default(present)
+        !$acc update self(pdata_neighbor(nparcelsLocalActive), &
+        !$acc             Cidx(nparcelsLocalActive))
```

Comment:
I don't understand why you are updating the CPU here.  The pdata_neighbour and Cidx array are never used by the CPU.

Reply:
I think I understand the need to change the other loops.  I don't understand the need to change this loop.  The values are never used by the CPU.  Have you confirmed that changes to this particular loop are necessary for correctness, or are you just changing all instances of acc serial?

Reply:
Aha, I see. I just changed all the instances of `acc serial` without a `do` loop associated. I could revert this one to the original `acc serial` and double check if it matters.

Reply:
Ok, so reverting this line back to `acc serial` seems ok (though I do not understand why it works since there is no `do` loop here). I am trying `acc kernel` now and see if I could avoid the use of `acc update` for the other places.

Reply:
@johnmauff Actually `acc kernels` also works and I think it makes more sense than using `acc update`. I just pushed those changes.

Reply:
Thanks for updating the code with the acc kernels